### PR TITLE
List documents and get folder content

### PR DESCRIPTION
**Pull request by blaskodaniel**

### CODE REVIEW

File: src/components/navbar.tsx.rej [hunk 1]
```diff
@@ -0,0 +1,61 @@
+diff a/src/components/navbar.tsx b/src/components/navbar.tsx	(rejected hunks)
```
Comment (B3zo0) [inline src/components/navbar.tsx.rej]: I'm pretty sure you don't want to commit this file. 😄

File: package.json [hunk 2]
```diff
@@ -74,7 +76,8 @@
     "@sensenet/default-content-types": "^1.2.2",
     "@sensenet/list-controls-react": "^1.3.8",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "react-responsive": "^7.0.0"
```
Comment (B3zo0) [inline package.json]: You are not using this library anywhere. Are you going to? if not then it is not needed.

File: src/components/mainpanel.tsx [hunk 3]
```diff
@@ -1,43 +1,49 @@
 import React, { useEffect, useState } from 'react'
+import clsx from 'clsx'
```
Comment (B3zo0) [inline src/components/mainpanel.tsx]: I see that this package is used by material-ui but you should add this to package.json as a dependency.

Comment (blaskodaniel) [reply src/components/mainpanel.tsx]: I fixed it

File: package.json (hunk 2 above)
Comment (blaskodaniel) [reply package.json]: Should I move it to "devDependencies" section?

**Pull request merged**